src/annotator/plugin/document.coffee

```diff
@@ -185,9 +186,7 @@ module.exports = class Document extends Plugin
 
   # Hack to get a absolute url from a possibly relative one
   _absoluteUrl: (url) ->
-    d = @document.createElement('a')
-    d.href = url
-    d.href
+    normalizeURI(url, @baseURI)
```

Comment:
The `href` property of an `HTMLAnchorElement` returns an absolute URL, resolved against the document's base URL, so it does the same thing as the `normalizeURI` function.